examples/tensorrt_llm/common/parser.py

```diff
@@ -40,6 +40,13 @@ def __init__(
         self.kv_cache_config = kv_cache_config
         self.extra_args = kwargs
 
+        # Hardcoded to skip tokenizer init for now.
+        # We will handle the tokenization/detokenization
+        #  in the base engine.
+        if "skip_tokenizer_init" in kwargs:
+            kwargs.pop("skip_tokenizer_init")
+        self.skip_tokenizer_init = True
```

Comment:
nit: we don't have to pop the kwargs - we can just set it to the value? (keep comment - but just set it instead of popping it and then setting it?)

Reply:
I wanted to remove it from the extra_args that get fed to the engine if a user has provided it.